data/World/Overworld.json

```diff
@@ -1446,13 +1615,44 @@
             "GC Pot Freestanding PoH": "
                 is_child and 'Goron City Child Fire' and
                 (Bombs or (Progressive_Strength_Upgrade and logic_goron_city_pot_with_strength) or (has_bombchus and logic_goron_city_pot))",
+            "GC Spinning Pot Bomb Drop 1": "
+            is_child and 'Goron City Child Fire' and
+            (Bombs or (Progressive_Strength_Upgrade and logic_goron_city_pot_with_strength) or (has_bombchus and logic_goron_city_pot))",
+            "GC Spinning Pot Bomb Drop 2": "
+            is_child and 'Goron City Child Fire' and
+            (Bombs or (Progressive_Strength_Upgrade and logic_goron_city_pot_with_strength) or (has_bombchus and logic_goron_city_pot))",
+            "GC Spinning Pot Bomb Drop 3": "
+            is_child and 'Goron City Child Fire' and
+            (Bombs or (Progressive_Strength_Upgrade and logic_goron_city_pot_with_strength) or (has_bombchus and logic_goron_city_pot))",
+            "GC Spinning Pot Rupee Drop 1": "
+            is_child and 'Goron City Child Fire' and
+            (Bombs or (Progressive_Strength_Upgrade and logic_goron_city_pot_with_strength) or (has_bombchus and logic_goron_city_pot))",
+            "GC Spinning Pot Rupee Drop 2": "
+            is_child and 'Goron City Child Fire' and
+            (Bombs or (Progressive_Strength_Upgrade and logic_goron_city_pot_with_strength) or (has_bombchus and logic_goron_city_pot))",
+            "GC Spinning Pot Rupee Drop 3": "
+            is_child and 'Goron City Child Fire' and
+            (Bombs or (Progressive_Strength_Upgrade and logic_goron_city_pot_with_strength) or (has_bombchus and logic_goron_city_pot))",
+            "GC Spinning Pot PoH Drop Rupee 1": "
+            is_child and 'Goron City Child Fire' and
+            (Bombs or (Progressive_Strength_Upgrade and logic_goron_city_pot_with_strength) or (has_bombchus and logic_goron_city_pot))",
+            "GC Spinning Pot PoH Drop Rupee 2": "
+            is_child and 'Goron City Child Fire' and
+            (Bombs or (Progressive_Strength_Upgrade and logic_goron_city_pot_with_strength) or (has_bombchus and logic_goron_city_pot))",
```

Comment:
This is a lot of repetition.

Reply:
Would you suggest a logic helper such as "can_blow_up_goron_pot"?

Reply:
Honestly, a spinning pot reward region?

Reply:
I believe logic helpers come with some amount of optimization so performance would not be an issue. I still think a region would be better than a logic helper that's only relevant for a single region.